### DISCUSSION TITLE
ScriptReference/EditorUserBuildSettings-webGLBuildSubtarget

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#512] [Notes]
# ScriptReference/EditorUserBuildSettings-webGLBuildSubtarget

https://docs.unity3d.com/ScriptReference/EditorUserBuildSettings-webGLBuildSubtarget.html

**Comment by radiatoryang** [comment]
This doesn't actually work anymore. You have to use the BuildOptions:

```
BuildPlayerOptions buildPlayerOptions = new BuildPlayerOptions();
buildPlayerOptions.subtarget = (int)WebGLTextureSubtarget.ASTC;
```
